warehouse/query-core/src/main/java/datawave/query/transformer/UniqueTransform.java

```diff
@@ -46,7 +47,14 @@ public class UniqueTransform extends DocumentTransform.DefaultDocumentTransform
     private UniqueFields uniqueFields;
     private Multimap<String,String> modelMapping;
 
-    public UniqueTransform(UniqueFields uniqueFields) {
+    /**
+     * Length of time in milliseconds that a client will wait while results are collected. If a full page is not collected before the timeout, a blank page will
+     * be returned to signal the request is still in progress.
+     */
+    private final long queryExecutionForPageTimeout;
+
+    public UniqueTransform(UniqueFields uniqueFields, long queryExecutionForPageTimeout) {
```

Comment:
Documentation would be nice, with a brief explanation for how the added `queryExecutionForPageTimeout` parameter changes the behavior of the transform

Reply:
done